### PR TITLE
Add the issue checker to enterprise scripts folder

### PR DESCRIPTION
**Pull request by Kingdutch**
This script looks at the extra.patches entry in a composer.json file and parses patches in the form of "issuenr - human readable description" to retrieve the issue status and display it in a pretty table.

screen shot 2017-08-09 at 15 53 10

### CODE REVIEW

**Comment by Kingdutch** [comment]
screen shot 2017-08-21 at 14 27 22

The improved version adds support for GitHub PRs

**Review by bramtenhove — CHANGES_REQUESTED** [review]
The index.js file has some indentation issues (around line 10) and extra blank lines around line 23.

**Comment by Kingdutch** [comment]
I've fixed the whitespace issues, improved the usage help text and made the `composer.json` argument optional in the process.

File: issue_checker/readme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Issue Checker
+
+The issue checker takes a composer.json file and checks the issue status of any 
+patches that are applied in it.
+
+## Usage
+
+```
+/path/to/issue_checker/index.js /path/to/composer.json
```
Comment (bramtenhove) [inline issue_checker/readme.md]: @Kingdutch the composer.json was not needed right? Can you update it in the readme?

Comment (Kingdutch) [reply issue_checker/readme.md]: You're right, I've fixed it

**Review by bramtenhove — APPROVED** [review]

**Pull request merged**